### PR TITLE
[LTS] API Update

### LINKED ISSUE #428: [LTS] API Updated
https://confluence.tsi-dev.otc-service.com/display/DR/RM-3680-LTS-API?sortBy=date&sortOrder=descending

### PR DESCRIPTION
**Pull request by Aloento**
fix #428

### CODE REVIEW

**Review by artem-lifshits — DISMISSED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by artem-lifshits — DISMISSED** [review]

File: openstack/lts/v2/transfers/DeleteTransfer.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package transfers
+
+import "github.com/opentelekomcloud/gophertelekomcloud"
+
+func DeleteTransfer(client *golangsdk.ServiceClient, transferId string) (err error) {
+	// DELETE /v2/{project_id}/transfers
+	_, err = client.Delete(client.ServiceURL("transfers")+"?log_transfer_id="+transferId, nil)
```
Comment (artem-lifshits) [inline openstack/lts/v2/transfers/DeleteTransfer.go]: 200 returned here with body:
https://docs.otc.t-systems.com/api/lts/lts_api_0030.html

File: openstack/lts/v2/transfers/UpdateTransfer.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+	//
+	// Enumerated values:
+	//
+	// min
+	// hour
+	ObsPeriodUnit string `json:"obs_period_unit"`
+	// OBS bucket path, which is the log transfer destination.
+	ObsTransferPath string `json:"obs_transfer_path,omitempty"`
+	// Enterprise project ID of an OBS transfer task.
+	ObsEpsId string `json:"obs_eps_id,omitempty"`
```
Comment (artem-lifshits) [inline openstack/lts/v2/transfers/UpdateTransfer.go]: Please remove `obs_eps_id`

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

File: openstack/common/pointerto/pointers.go [hunk 3]
```diff
@@ -5,6 +5,10 @@ func Int(src int) *int {
 	return &src
 }
 
+func Int32(src int32) *int32 {
```
Comment (artem-lifshits) [inline openstack/common/pointerto/pointers.go]: But it's not even used?..

File: openstack/lts/v2/streams/ListLogs.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+func ListLogs(client *golangsdk.ServiceClient, opts ListLogsOpts) (*ListLogsResponse, error) {
+	b, err := build.RequestBody(opts, "")
+	if err != nil {
+		return nil, err
+	}
+
+	// POST /v2/{project_id}/groups/{log_group_id}/streams/{log_stream_id}/content/query
+	raw, err := client.Post(client.ServiceURL("groups", opts.GroupId, "streams", opts.StreamId, "content", "query"), b, nil, nil)
```
Comment (artem-lifshits) [inline openstack/lts/v2/streams/ListLogs.go]: Status code 200 
https://docs.otc.t-systems.com/api/lts/lts_api_0028.html

File: acceptance/openstack/lts/v2/lts_test.go [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+			GroupId:  created,
+			StreamId: stream,
+		})
+		th.AssertNoErr(t, err)
+	})
+
+	slist, err := streams.ListLogStream(client, created)
+	th.AssertNoErr(t, err)
+	th.AssertEquals(t, true, len(slist) > 0)
+	tools.PrintResource(t, slist)
```
Comment (artem-lifshits) [inline acceptance/openstack/lts/v2/lts_test.go:64]: No tests for `transfers` section

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

File: acceptance/openstack/lts/v2/lts_test.go [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/lts/v2/groups"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/lts/v2/streams"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestLtsLifecycle(t *testing.T) {
```
Comment (artem-lifshits) [inline acceptance/openstack/lts/v2/lts_test.go:16]: Current test is failing:
```
=== RUN   TestLtsLifecycle
    lts_test.go:36: Failure in lts_test.go, line 36: expected 3 but got 3
    lts_test.go:28: Failure in lts_test.go, line 28: unexpected error "Bad request with: [DELETE https://lts.eu-de.otc.t-systems.com/v2/a720688fb87f4575a4c000d818061eae/groups/b629e2b1-ee94-4cdd-8f4b-0712f95a8e0c], error message: {\"error_msg\":\"Orchestration error: Invalid header parameter: Content-Type, required\",\"error_code\":\"APIGW.0106\",\"request_id\":\"aee8279c013c5ca4d784549894aaaa31\"}\n"
--- FAIL: TestLtsLifecycle (1.40s)

FAIL

Process finished with the exit code 1
```

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

File: openstack/common/pointerto/pointers.go (hunk 3 above)
Comment (Aloento) [reply openstack/common/pointerto/pointers.go]: yes hhh

File: acceptance/openstack/lts/v2/lts_test.go (hunk 6 above)
Comment (Aloento) [reply acceptance/openstack/lts/v2/lts_test.go:16]: done

File: openstack/lts/v2/streams/ListLogs.go (hunk 4 above)
Comment (Aloento) [reply openstack/lts/v2/streams/ListLogs.go]: done

File: openstack/lts/v2/transfers/UpdateTransfer.go (hunk 2 above)
Comment (Aloento) [reply openstack/lts/v2/transfers/UpdateTransfer.go]: done

File: openstack/lts/v2/transfers/DeleteTransfer.go (hunk 1 above)
Comment (Aloento) [reply openstack/lts/v2/transfers/DeleteTransfer.go]: done

**Review by artem-lifshits — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**